src/app/machines/views/MachineList/MachineList.tsx

```diff
@@ -37,6 +37,33 @@ const parseFilters = (filters: Filters): FetchFilters => {
   return fetchFilters;
 };
 
+const ErrorsNotification = ({
```

Comment:
In MAAS we have a one component per file rule to help keep file lengths down as well as make it easier to test and also encourage reusing components.

Reply:
As a follow up, I added an eslint rule so that we don't have to rely on manual checks that we follow this. https://github.com/canonical/maas-ui/pull/4340